clients.json

```diff
@@ -1554,6 +1554,15 @@
     "description": "A Xojo library to connect to a Redis server.",
     "authors": ["kemtekinay"],
     "active": true
+  },
+  
+  {
+    "name": "song-yuan",
+    "language": "PHP",
+    "repository": "https://github.com/song-yuan/redis-doc",
```

Comment:
Sorry. Looks like it is a wrong link, it refers to forked redis-doc repo. Could you please double check.